GeoMesa_Feature_List.md

```diff
@@ -0,0 +1,100 @@
+  - 'Tube selection' (space/time correlated queries)
+    - Pointer: [geomesa 'tube' queries](geomesa/geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/process/tube/)
+    - This is a pretty sophisticated query mechanism. The basic idea is that, given a collection of points (with associated times), you should be able to return similar collections of points (in terms of where the lines connecting said points exist). Constraints on the query include the size of the spatial and temporal buffers (this is the sense in which we're dealing with 'tubes') and maximum speed attained by the entity whose points make up a given trajectory. Read more here: http://www.geomesa.org/documentation/tutorials/geomesa-tubeselect.html
+  - Proximity Search
+    - Pointer: [ProximitySearchProcess.scala](https://github.com/locationtech/geomesa/blob/b7056fae4988ef524913bf3dc33d9ff2a3476b09/geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/process/proximity/ProximitySearchProcess.scala)
+    - Given a set of vectors to search through and a set of vectors to establish proximity, return the members of the former set which lie within the (specified) proximity of members of the latter set
+  - Query against an accumulo GeoMesa store
+    - Pointer: [QueryProcess.scala](https://github.com/locationtech/geomesa/blob/b7056fae4988ef524913bf3dc33d9ff2a3476b09/geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/process/query/QueryProcess.scala)
+    - Takes advantage of accumulo optimization to carry out geomesa queries
+  - Find the K nearest neighbors to a given point
+    - Pointer: [KNearestNeighborSearchProcess.scala](https://github.com/locationtech/geomesa/blob/b7056fae4988ef524913bf3dc33d9ff2a3476b09/geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/process/knn/KNearestNeighborSearchProcess.scala)
+  - Identify unique values for an attribute in results of a CQL query
+    - Pointer: [UniqueProcess.scala](https://github.com/locationtech/geomesa/blob/b7056fae4988ef524913bf3dc33d9ff2a3476b09/geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/process/unique/UniqueProcess.scala)
+  - Convert points to lines
+    - Pointer: [Point2PointProcess.scala](https://github.com/locationtech/geomesa/blob/b7056fae4988ef524913bf3dc33d9ff2a3476b09/geomesa-process/src/main/scala/org/locationtech/geomesa/process/Point2PointProcess.scala)
+    - Convert a collection of points into a collection of line segments given a middle term parameter. Optionally break on the day of occurrence. This feature isn't really advertised.
+
+
+## Output
+- geomesa-accumulo
+  - A reader for directly querying a datastore in java/scala
```

Comment:
as more explanation - the feature reader/feature source/feature writer implement the OGC standard geotools framework.